src/TSMapEditor/Models/TriggerCondition.cs

```diff
@@ -8,11 +8,32 @@ namespace TSMapEditor.Models
     /// </summary>
     public class TriggerCondition : ICloneable
     {
-        public const int INI_VALUE_COUNT = 3;
+        public const int DEF_PARAM_COUNT = 2;
+        public const int MAX_PARAM_COUNT = 3;
+
+        public TriggerCondition()
+        {
+            for (int i = 0; i < Parameters.Length - 1; i++)
+                Parameters[i] = "0";
+
+            Parameters[MAX_PARAM_COUNT - 1] = string.Empty;
+        }
 
         public int ConditionIndex { get; set; }
-        public int Parameter1 { get; set; }
-        public int Parameter2 { get; set; }
+
+        public string[] Parameters { get; private set; } = new string[MAX_PARAM_COUNT];
+
+        public string ParamToString(int index)
+        {
+            if (string.IsNullOrWhiteSpace(Parameters[index]))
+            {
+                if (index == MAX_PARAM_COUNT - 1)
+                    return string.Empty;
+                return "0";
```

Comment:
Leave a blank line between the return statements. In most cases, having a blank line after a braceless `if` looks clearer IMO.

```suggestion

                return "0";
```